src/observable/bindCallback.ts

```diff
@@ -9,6 +9,24 @@ import {AsyncSubject} from '../subject/AsyncSubject';
 export class BoundCallbackObservable<T> extends Observable<T> {
   subject: AsyncSubject<T>;
 
+  /* tslint:disable:max-line-length */
+  static create<TResult>(callbackFunc: (callback: (result: TResult) => any) => any, selector?: void, scheduler?: Scheduler): () => Observable<TResult>;
```

Comment:
Would you elaborate bit more? does that mean this signature update is not even required by `Observable` should have correct signature?